pandas/tests/extension/arrow/bool.py

```diff
@@ -9,10 +9,11 @@
 import itertools
 
 import numpy as np
```

Comment:
you can use

pytest.importorskip here

Reply:
pytest.importorskip have to be placed before exact import, but I don't think so it's possible now. Isort won't allow to do that.